build/Targets/Settings.targets

```diff
@@ -135,6 +135,8 @@
   <PropertyGroup>
     <IsVSBeforeDev15Preview4 Condition="'$(OS)' == 'Windows_NT' AND '$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\15.0@InstallDir)' != ''">true</IsVSBeforeDev15Preview4>
     <IsVSBeforeDev15Preview4 Condition="'$(OS)' == 'Windows_NT' AND '$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\VisualStudio\15.0@InstallDir)' != ''">true</IsVSBeforeDev15Preview4>
+    <IsVSBeforeDev15Preview4 Condition="'$(IsVSBeforeDev15Preview4)' == 'true' AND Exists('$(MSBuildBinPath)\..\..\..\Common7\IDE\devenv.isolation.ini')">false</IsVSBeforeDev15Preview4>
```

Comment:
It requires VSCOMNTOOLS (or VS150COMNTOOLS) to be defined, which does not exist on Preview 4 unless you are running from the Developer Command Prompt.

I don't believe it is safe to assume that the user is running from the Developer Command Prompt, as they may have manually added MSBuild to their path instead.